src/context/CartProvider.jsx

```diff
@@ -119,12 +119,43 @@ export const CartProvider = ({ children }) => {
     }
   };
 
+  const cartCheckout = async () => {
+    try {
+      const response = await axiosInstance.post(`orders`, {
+        order: {
+          credit_card: {
+            card_number: "4242424242424242",
+            exp_month: 8,
```

Comment:
Sugiero usar un mes más a futuro para evitar que devuelva un error de que es invalido (por ejemplo ahora que estamos en septiembre)

```suggestion
            exp_month: 12,
```